docs/catalog/ai/leverage-sustainable-regions.md

```diff
@@ -0,0 +1,39 @@
+---
+version: 1.0
+submitted_by: navveenb
+published_date: tbd
+category: ai
+tags: 
+ - ai
+ - machine-learning
+ - role:data-scientist
+ - size:small
+---
+
+# Use sustainable regions for ML training 
+
+## Description
+Training an AI model has a significant carbon footprint. Leverage
+a region which offers carbon-free energy for running your ML training jobs.
+
+
+## Solution
+Use a cloud region which offers carbon-free energy for running your ML training workloads.
+
+
+## SCI Impact
+`SCI = (E * I) + M per R`
+
+For the SCI equation, using a sustainable region for ML training would impact the following:
+- 'E': Having a carbon free region for ML training, would reduce the carbon emissions of ML applications, decreasing the E number.
+
+## Assumptions
+None 
+
+## Considerations
+None
```

Comment:
Costs of resources across different regions are not the same so you should add a consideration of the financial impact this decision will have. Same with potential increase to data transfer latency if regions are far apart.

Reply:
Good point. I have added this.